Gemfile

```diff
@@ -174,7 +174,7 @@ group :test, :development do
   gem 'rspec-activemodel-mocks', '~> 1.0.0'
   gem 'rspec-rails', '~> 3.5.0'
   gem 'pry', '~> 0.10.0'
-  gem 'pry-debugger', '~> 0.2.0', :platforms => :ruby_19
+  gem 'pry-byebug', '~> 3.4.0'
```

Comment:
> Locked to 3.4.x to retain support for Ruby 2.0 and 2.1 (removed from
> byebug in version 9.1.0)

9.1.0 a typo?

Reply:
no, I think that's correct, just confusingly stated because of the similar names:
https://github.com/mysociety/alaveteli/pull/4348/files#diff-e79a60dc6b85309ae70a6ea8261eaf95R256

\*tries to think of better phrasing\*

Reply:
"(removed from the underlying byebug dependency in version 9.1.0)"? (or similar)

Reply:
and/or a link to their ticket for that github.com/deivid-rodriguez/pry-byebug/issues/123